ocaml/xapi/helpers.ml

```diff
@@ -63,10 +63,16 @@ let get_pool ~__context = List.hd (Db.Pool.get_all ~__context)
 let get_master ~__context =
   Db.Pool.get_master ~__context ~self:(get_pool ~__context)
 
-let get_management_iface_is_connected ~__context =
+let get_iface_is_connected ~__context iface =
```

Comment:
Note that this is checking whether at least one of the physical interface on the given bridge is connected, so it's a little confusing that this function talks about interfaces. Please add a comment above the function, and replace `iface` by `bridge`.

Reply:
I think I was slightly confused here because I didn't expect a single physical interface to always have a bridge, though in that case it would be a bridge with 1 interface, so should be fine?

Reply:
The way networking is set up is that each physical interface is always on its own bridge, unless it is bonded and there are multiple physical interface on a single bridge. And the IP addresses of dom0 are always on bridges.